src/cascade/dismod/serialize.py

```diff
@@ -292,21 +295,23 @@ def make_avgint_table(context, integrand_id_func):
         if integrand.age_ranges is not None and integrand.time_ranges is not None:
             for age_lower, age_upper in integrand.age_ranges:
                 for time_lower, time_upper in integrand.time_ranges:
-                    rows.append(
-                        {
-                            "integrand_id": integrand_id_func(integrand.name),
-                            "age_lower": age_lower,
-                            "age_upper": age_upper,
-                            "time_lower": time_lower,
-                            "time_upper": time_upper,
-                            # Assuming using the first set of weights, which is constant.
-                            "weight_id": 0,
-                            # Assumes one location_id.
-                            "node_id": 0,
-                        }
-                    )
+                    for sex in [-0.5, 0.5]:
+                        rows.append(
+                            {
+                                "integrand_id": integrand_id_func(integrand.name),
+                                "age_lower": age_lower,
+                                "age_upper": age_upper,
+                                "time_lower": time_lower,
+                                "time_upper": time_upper,
+                                # Assuming using the first set of weights, which is constant.
+                                "weight_id": 0,
+                                # Assumes one location_id.
+                                "node_id": 0,
+                                "x_sex": sex,
```

Comment:
At this point in the computation, it has to be there in the IHME data... I wonder whether I can make an "add_covariate" call that makes sense here. It would have to both make the column and add to the incoming data. Let me think on this one. good point.

Reply:
If it doesn't make sense then it doesn't and that's fine. I don't have enough of this system in my head yet to understand if/how hard this is.